composer.json

```diff
@@ -30,7 +30,7 @@
     "wpackagist-plugin/google-apps-login": "3.4.1",
     "wpackagist-plugin/google-sitemap-generator": "4.*",
     "wpackagist-plugin/redirection": "4.*",
-    "wpackagist-plugin/timber-library": "1.9.4",
+    "wpackagist-plugin/timber-library": "1.16.0",
```

Comment:
Do you think it's safe to use `1.16.*` or is there a risk of a patch version to break something?

Reply:
To be safe, I think we should go with specific minor version. (at leas)